api/src/pcapi/core/users/young_status.py

```diff
@@ -53,6 +55,27 @@ class Suspended(YoungStatus):
     status_type: YoungStatusType = YoungStatusType.SUSPENDED
 
 
+@dataclasses.dataclass
+class UserSubscriptionState:
+    # fraud_status holds the user status relative to its fraud checks. It is mainly used in the admin interface.
+    fraud_status: subscription_models.SubscriptionItemStatus
+
+    # next_step holds the next step to be done by the user to complete its subscription.
+    next_step: subscription_models.SubscriptionStep | None
+
+    # young_status holds the user status relative to its subscription. It is mainly used in the frontend.
+    young_status: YoungStatus
+
+    # identity_fraud_check is the relevant identity fraud check used to calculate their status.
+    fraud_check: fraud_models.BeneficiaryFraudCheck | None = None  # identity fraud check (a renommer)
+
+    # is_activable is True if the user can be activated.
+    is_activable: bool = False
+
+    # subscription_message is the message to display to the user. It is mainly used in the frontend.
```

Comment:
Be careful : in the frontend, this message is displayed with higher priority than next_step call to action